app/components/sidebars/settings/index.js

```diff
@@ -29,6 +29,7 @@ function makeMapStateToProps() {
             theme: getTheme(state),
             isCustomStatusEnabled: customStatusEnabled,
             customStatus,
+            isCustomStatusExpired: isCustomStatusExpired(state, customStatus),
```

Comment:
why compute if `customStatusEnabled` is false?
```suggestion
            isCustomStatusExpired: customStatusEnabled ? isCustomStatusExpired(state, customStatus) : false,
```